gpflow/config/__config__.py

```diff
@@ -1,3 +1,40 @@
+"""
+This is a private module that manages GPflow configuration.
+
+The module provides functions to modify default settings of GPflow, such as:
+the standard float precision and integer type, the positive transformation
+type, a value for a minimum shift from zero for the positive transformation,
+an output format for the summary printing.
+
+The module holds global configuration :class:`Config` variable that stores all
+setting values.
+
+The environment variables is an alternative way for changing default GPflow
+configuration.
+
+.. warning::
+    The user has to set environment variables before running python
+    interpreter to modify the configuration.
+
+Full set of environment variables and available options:
+    ```
+    GPFLOW_INT: "int16", "int32", or "int64"
```

Comment:
can these be any type from https://www.tensorflow.org/api_docs/python/tf/dtypes/DType ?

Reply:
It is a section about environment variable and you will pass a string there. Although, in a setter, you are allowed to pass TensorFlow and NumPy types.